README.md

```diff
@@ -33,6 +33,7 @@ Slevomat Coding Standard for [PHP_CodeSniffer](https://github.com/squizlabs/PHP_
  - [SlevomatCodingStandard.Arrays.DisallowImplicitArrayCreation](doc/arrays.md#slevomatcodingstandardarraysdisallowimplicitarraycreation)
  - [SlevomatCodingStandard.Arrays.DisallowPartiallyKeyed](doc/arrays.md#slevomatcodingstandardarraysdisallowpartiallykeyed) 🚧
  - [SlevomatCodingStandard.Arrays.MultiLineArrayEndBracketPlacement](doc/arrays.md#slevomatcodingstandardarraysmultilinearrayendbracketplacement-) 🔧
+ - [SlevomatCodingStandard.Arrays.NoSpaceBetweenArrayNameAndBracketsSniff.php](doc/arrays.md#slevomatcodingstandardarraysnospacebetweenarraynameandbrackets) 🔧
```

Comment:
Tady to vypadá, že pak chybí ta dokumentace :)

A v odkazu afaik má být na konci ta pomlčka, protože ta představuje ten klíč, který v dokumentaci bude v nadpisu.